Add warning about changing order status after filtering orders in ug (#231)
AY2425S1-CS2103T-F14a-2/tp: docs/UserGuide.md

</box>

<box type="warning">

##### Warning

Filtering the order list by status and changing an order's status will cause the order to not be displayed under the filtered list.

> 

Reviewer: Maybe can give example? Like for example filter to completed and then unmark it or something like that

Reviewer 2: Then can move all the warning statements into a separate warning box? Other commands just have warnings as part of the "constraints" box

Reviewer: this is not like giving warning in the GUI. This is warning to the user

Author: > Maybe can give example? Like for example filter to completed and then unmark it or something like that

Will add example.

> Then can move all the warning statements into a separate warning box? Other commands just have warnings as part of the "constraints" box

Like foochao said, this is a warning in the user guide rather than GUI. It's mainly to let the reader know that this is a thing.

</box>

<box type="warning">

##### Warning

> Filtering the order list by status and changing an order's status will cause the order to not be displayed under the filtered list.<br>

Reviewer 3: ```suggestion
Filtering the order list by status and then changing an order's status will result in the order no longer appearing under the filtered list. <br>
```

> e.g. `filterorder pending` followed by marking the first order in the filtered list as completed will cause that completed order to not be displayed in the current filtered list. 

Reviewer 3: ```suggestion
e.g. using `filterorder pending` followed by marking the first order in the filtered list as completed will remove that order from the current filtered list.
```
